libs/design/src/molecules/feature/feature.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { DaffFeatureComponent } from './feature.component';
+
+describe('DaffFeatureComponent', () => {
+  let component: DaffFeatureComponent;
+  let fixture: ComponentFixture<DaffFeatureComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        DaffFeatureComponent
+      ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(DaffFeatureComponent);
+    component = fixture.componentInstance;
+
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Needs a test about adding a class to host.

Reply:
Resolved in 41f8ddf